### ISSUE TITLE
warning using deprecated function dplyr::select_ in ggsurvplot_combine

### ISSUE THREAD

**Issue by travismark** [kassambara/survminer#578]
### Expected behavior 

ggsurvplot_combine runs without warnings

### Actual behavior

getting a "select_" deprecation warning while building the survival table

### Steps to reproduce the problem

```{R}
library(survival)
library(survminer)
data("lung")
fit1 <- survfit(Surv(time, status) ~ sex, data = lung)
fit2 <- survfit(Surv(time, status) ~ sex, data = lung, start.time=250)
p <- ggsurvplot_combine(list(
    original=fit1, conditional=fit2
), data = lung)
```

> Warning messages:
> 1: Problem while computing `survtable = purrr::map2(...)`.
> ℹ `select_()` was deprecated in dplyr 0.7.0. Please use `select()` instead. 

### `session_info()`

````{R}
> sessionInfo()
R version 3.6.2 (2019-12-12)
Platform: x86_64-apple-darwin19.0.0 (64-bit)
Running under: macOS  10.16

Matrix products: default
BLAS/LAPACK: /Users/tbaer/.brew/Cellar/openblas/0.3.7/lib/libopenblasp-r0.3.7.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

other attached packages:
[1] survminer_0.4.6 ggpubr_0.4.0    ggplot2_3.3.5   survival_3.1-12

loaded via a namespace (and not attached):
 [1] zoo_1.8-9         xfun_0.13         tidyselect_1.1.1  purrr_0.3.4
 [5] splines_3.6.2     haven_2.4.3       lattice_0.20-41   carData_3.0-4
 [9] colorspace_1.4-1  vctrs_0.3.8       generics_0.0.2    utf8_1.1.4
[13] survMisc_0.5.5    rlang_1.0.1       pillar_1.7.0      foreign_0.8-76
[17] glue_1.6.1        withr_2.4.3       DBI_1.1.0         readxl_1.3.1
[21] lifecycle_1.0.1   munsell_0.5.0     ggsignif_0.6.3    gtable_0.3.0
[25] cellranger_1.1.0  zip_2.2.0         labeling_0.3      knitr_1.28
[29] rio_0.5.27        forcats_0.5.1     curl_4.3          fansi_0.4.1
[33] broom_0.7.10      Rcpp_1.0.4.6      xtable_1.8-4      backports_1.1.6
[37] scales_1.1.0      abind_1.4-5       farver_2.0.3      km.ci_0.5-2
[41] gridExtra_2.3     hms_1.1.1         stringi_1.4.6     openxlsx_4.2.4
[45] rstatix_0.7.0     dplyr_1.0.8       KMsurv_0.1-5      grid_3.6.2
[49] cli_3.1.0         tools_3.6.2       magrittr_2.0.2    tibble_3.1.6
[53] crayon_1.5.0      car_3.0-11        tidyr_1.2.0       pkgconfig_2.0.3
[57] ellipsis_0.3.2    Matrix_1.2-18     data.table_1.14.2 assertthat_0.2.1
[61] R6_2.4.1          compiler_3.6.2
````

**Comment by dpprdan** [comment]
See also #509

https://github.com/kassambara/survminer/search?q=select_

**Comment by kassambara** [comment]
fixed in #579 and #665
